packages/axios-odata-client/src/AxiosODataClientError.ts

```diff
@@ -0,0 +1,10 @@
+import { AxiosError } from "axios";
+
+export class AxiosODataClientError extends Error {
+  name = "AxiosODataClientError";
+  isAxiosOdataClientError = true;
```

Comment:
do you think that this information "isAxisoOdataClientError" has any value? 
Should it be removed?

Or do you want to go for a type? Would be easier to determine which kind of error you have: either axios or Jquery..